modules/dashboard/components/multi-labels-select.tsx

```
@@ -0,0 +1,134 @@
+                    variant="secondary"
+                    className="text-xs"
+                    style={{
+                      backgroundColor: `${label.color}20`,
+                      color: label.color,
+                      borderColor: label.color,
+                    }}
+                  >
+                    {label.name}
+                    <span
+                      tabIndex={0}
+                      role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)